src/sagemaker_containers/environment.py

```diff
@@ -34,7 +38,7 @@
 JOB_NAME_ENV = 'JOB_NAME'  # type: str
 USE_NGINX_ENV = 'SAGEMAKER_USE_NGINX'  # type: str
 
-BASE_PATH = os.environ.get(BASE_PATH_ENV, os.path.join('/opt', 'ml'))  # type: str
+BASE_PATH = os.environ.get(BASE_PATH_ENV, os.path.join(os.sep, 'opt', 'ml'))  # type: str
```

Comment:
if first arg to join has '/' it's ok, but if later ones do then it's trouble. looks like os.sep here is just to avoid platform-specific separator. but '/' should work fine on all platforms and is more readable

Reply:
I will revert.